### PR TITLE
Update project file and shared schemes to allow both a Carthage build and Xcode workspace build.

### PR DESCRIPTION
**Pull request by darinkrauss**

### CODE REVIEW

File: LoopKit.xcodeproj/xcshareddata/xcschemes/Shared watchOS.xcscheme [hunk 1]
```diff
@@ -14,9 +14,9 @@
             buildForAnalyzing = "YES">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "43BA7153201E484D0058961E"
-               BuildableName = "LoopKitUI.framework"
-               BlueprintName = "LoopKitUI"
+               BlueprintIdentifier = "A9E6758022713F4700E25293"
+               BuildableName = "LoopKit.framework"
+               BlueprintName = "LoopKit watchOS"
```
Comment (ps2) [inline LoopKit.xcodeproj/xcshareddata/xcschemes/Shared watchOS.xcscheme]: The pattern in loop is with a dash. I.e. `LoopKit-watchOS`.  Not strictly necessary to change, but if we want to keep a pattern.

**Review by ps2 — APPROVED** [review]
LGTM, just one naming suggestion, if you agree.

**Comment by darinkrauss** [comment]
@ps2 I updated all of the targets in the affected repos to use the dash rather than space before `watchOS` (eg. `LoopKit-watchOS`). I also updated the schemes to be `Shared` and `Shared-watchOS` to match.

**Review by ps2 — APPROVED** [review]

**Comment by darinkrauss** [comment]
[Create a repo and workspace for building Tidepool Loop](https://trello.com/c/or5FC9tm/589-create-a-repo-and-workspace-for-building-tidepool-loop)

**Pull request merged**